### PR TITLE
fix(mojaloop/#2883): resolve integration tests being broken due to import

### PR DESCRIPTION
**Pull request by kleyow**
fix(mojaloop/#2883): resolve integration tests being broken due to import - [https://github.com/mojaloop/project/issues/2883](https://github.com/mojaloop/project/issues/2883)

- Fix integration tests being broken due to import
- Update tests for new message content bodies

### CODE REVIEW

File: modules/outbound-command-event-handler/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "lint:fix": "eslint --ext .js,.ts src --color --fix",
     "copy-files": "npx copyfiles -u 1 src/**/*.yaml dist/",
     "test:unit": "jest --passWithNoTests --testMatch '**/test/unit/**/*.test.ts'",
-    "test:integration": "jest --passWithNoTests --testMatch '**/test/integration/**/*.test.ts'",
+    "test:integration": "jest --passWithNoTests --testMatch '**/test/integration/**/*.test.ts' --runInBand",
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/package.json:33]:
```suggestion
    "test:integration": "jest --runInBand --passWithNoTests --testMatch '**/test/integration/**/*.test.ts'",
```

**Review by vijayg10 — APPROVED** [review]

File: modules/outbound-command-event-handler/src/application/handler.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 'use strict';
 
-import { BulkTransactionAgg } from '@module-domain';
+import { BulkTransactionAgg } from '../domain';
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/src/application/handler.ts:27]: why the change?

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ import { CommandEvent, ICommandEventData, DomainEvent,
 } from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
 import { randomUUID } from "crypto";
 
-
+jest.setTimeout(20000)
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:46]: Please put a comment here as to why this is needed :D

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts [hunk 4]
```diff
@@ -158,7 +157,7 @@ describe("Tests for Outbound Command Event Handler", () => {
     const processSDKOutboundBulkRequestMessageObj = new ProcessSDKOutboundBulkRequestCmdEvt(sampleCommandEventData);
     await producer.sendCommandEvent(processSDKOutboundBulkRequestMessageObj);
 
-    await new Promise(resolve => setTimeout(resolve, 1000));
+    await new Promise(resolve => setTimeout(resolve, 2000));
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:160]: If we are going to hard-code a timeout on this, perhaps parameterise the timeout so we can change it in one place?

File: modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_accept_party_info_received.test.ts [hunk 5]
```diff
@@ -30,12 +30,18 @@ import { SDKSchemeAdapter } from "@mojaloop/api-snippets";
...
+  IRedisBulkTransactionStateRepoOptions,
+  IndividualTransferInternalState,
+  SDKOutboundBulkAcceptPartyInfoReceivedDmEvt,
+  ISDKOutboundBulkAcceptPartyInfoReceivedDmEvtData
+} from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
 import { randomUUID } from "crypto";
 
-import { BulkTransactionAgg, IndividualTransferEntity, IndividualTransferInternalState, IndividualTransferState } from '../../../src/domain'
-import { RedisBulkTransactionStateRepo, IRedisBulkTransactionStateRepoOptions } from '../../../src/infrastructure'
+import { BulkTransactionAgg } from '../../../src/domain'
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_accept_party_info_received.test.ts:44]: Why no alias?

File: modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_request_received.test.ts [hunk 6]
```diff
@@ -30,12 +30,16 @@ import { SDKSchemeAdapter } from "@mojaloop/api-snippets";
...
+  IKafkaEventProducerOptions,
+  ISDKOutboundBulkRequestReceivedDmEvtData,
+  SDKOutboundBulkRequestReceivedDmEvt,
+} from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
+
 import { randomUUID } from "crypto";
 
-import { BulkTransactionAgg, IndividualTransferEntity, IndividualTransferInternalState, IndividualTransferState } from '../../../src/domain'
-import { RedisBulkTransactionStateRepo, IRedisBulkTransactionStateRepoOptions } from '../../../src/infrastructure'
+import { BulkTransactionAgg } from '../../../src/domain'
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_request_received.test.ts:42]: Alias?

**Review by mdebarros — COMMENTED** [review]
Minor comments to address.

File: modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_accept_party_info_received.test.ts (hunk 5 above)
Comment (kleyow) [reply modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_accept_party_info_received.test.ts:44]: The test folder isn't set up to use modules, probably need to add it somewhere in tsconfig.

But mainly I feel very adversed in using the tsconfig alias's given that they result in errors even when there is no discernable circular reference.

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts (hunk 4 above)
Comment (kleyow) [reply modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:160]: done. added a top file `const`

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts (hunk 3 above)
Comment (kleyow) [reply modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:46]: done

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts [hunk 7]
```diff
@@ -43,8 +43,11 @@ import { CommandEvent, ICommandEventData, DomainEvent,
 } from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
 import { randomUUID } from "crypto";
 
+// Tests can timeout in a CI pipeline so giving it leeway
```
Comment (mdebarros) [inline modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:46]: Do we know why?

**Review by mdebarros — APPROVED** [review]
+1

File: modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts [hunk 8]
```diff
@@ -269,11 +271,9 @@ describe("Tests for Outbound Command Event Handler", () => {
     expect(filteredEvents.length).toBe(2);
     // Check the data contents for domain event
     expect(filteredEvents[0].getName()).toBe('PartyInfoRequestedDmEvt');
-    expect(JSON.parse(JSON.stringify(filteredEvents[0].getContent())).path).not.toContain('undefined');
```
Comment (sri-miriyala) [inline modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:272]: I originally had this undefined as the content had a field called path, that had /parties/MSISDN/<MSISDNID>/undefined. This test is to make sure the word "undefined" does not appear in the field value. @kleyow

Comment (kleyow) [reply modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts:272]: To my knowledge atm Sri, the content in this message contains no `path` field.
